samples/bluetooth/llpm/README.rst

```diff
@@ -0,0 +1,313 @@
+.. _ble_llpm:
+
+Bluetooth: LLPM
+###############
+
+The Bluetooth Low Latency Packet Mode (LLPM) sample uses the :ref:`latency_readme` to showcase the LLPM proprietary Bluetooth extension from Nordic Semiconductor.
+You can use it to determine the transmission latency of LLPM-enabled connections, or to compare with different connection parameters and check their influence on the results.
+
+
+Overview
+********
+
+The LLPM is designed time-sensitive transmissions, in which time synchronization is important and interface response time is critical for the user.
+For example, for reality headsets or gaming mouse and keyboard.
+
+See the following subsections for a description of the key LLPM elements.
+
+LLPM connection interval (1 ms)
+   The connection interval defines how often the devices must listen on the radio.
+   The LLPM introduces the possibility to reduce the connection interval below what is supported in BLE.
+   The lowest interval supported is 1 ms for one link.
```

Comment:
"The lowest supported connection interval is 1 ms"

Reply:
I agree. Let's rather have a link from the example to the service.